lib/deimos/utils/db_poller/base.rb

```diff
@@ -101,7 +113,7 @@ def process_batch_with_span(batch, status)
           begin
             span = Deimos.config.tracer&.start(
               'deimos-db-poller',
-              resource: @producer.class.name.gsub('::', '-')
+              resource: @resource_class.class.name.gsub('::', '-')
```

Comment:
`@resource_class` is already a class, so wouldn't this always just show `Class`?